modules/imgproc/src/filter.simd.hpp

```diff
@@ -465,6 +465,52 @@ struct RowVec_8u32s
     bool smallValues;
 };
 
+struct RowVec_8u32f
+{
+    RowVec_8u32f() {}
```

Comment:
Do we really need default ctor?

Reply:
I think default ctor is needed as an argument of `RowFilter( const Mat& _kernel, int _anchor, const VecOp& _vecOp=VecOp() )` where VecOp is `RowVec_8u32f`.